project/index.html

```diff
@@ -8,24 +8,59 @@
     <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
+    <div class="wrapper"> 
+        <header class="header-container">
+            <img src="image/logo.svg" alt="Logo" class="logo">
+            <nav class="header-nav">
+                <a href="#">Stays</a>
+                <a href="#">Attractions</a>
+                <a href="#"><img src="image/Night.svg" alt="Night mode"></a>
+                <a href="#"><img src="image/AccountCircle.svg" alt="Account"></a>
+            </nav>
+        </header>
+            <section class="container container-search">
+                <h1 class="title-section-search">Discover stays to live, work or just relax</h1>
+                <form action="/" method="GET" class="form-search">
+                    <div class="wrapper-label">
+                        <label for="destination-or-hotel">Your destination or hotel name</label>
+                        <label for="date-range">Check-in — Check-out</label>
+                    </div>
+                    <div class="wrapper-input">
+                        <input type="text" name="destination-or-hotel" placeholder="New York" id="destination-or-hotel">
+                        <input type="date" name="data-range" id="date-range" value="2021-05-11">
+                        <input type="text" name="accommodation" class="accommodation" placeholder="2 Adults — 0 Children — 1 Room">
+                        <input type="submit" class="btn-search" value="Search">
+                    </div>
+                </form>
+                <div class="app-gallery">
+                    <a href="https://play.google.com">
+                        <img src="image/google-play-badge.svg" alt="google">
+                    </a>
+                    <a href="https://apps.apple.com">
+                        <img src="image/App_Store_Badge.svg" alt="app-store">    
+                    </a>
```

Comment:
![image](https://user-images.githubusercontent.com/20189313/127403025-2b536548-1ea8-4155-9cc0-b1c2ea310b0d.png)

Reply:
pls fix it